src/webapi/Controllers/AuthorizationController.cs

```diff
@@ -82,18 +125,18 @@ public async Task<IActionResult> Exchange([FromForm] PasswordTokenRequest tokenR
                 .SetClaim(Claims.Email, await _userManager.GetEmailAsync(user))
                 .SetClaim(Claims.Name, $"{user.FirstName} {user.LastName}")
                 .SetClaims(Claims.Role, (await _userManager.GetRolesAsync(user)).ToImmutableArray());
-
+        tokenRequest.Scopes = "offline_access";
```

Comment:
What does this do? We now have a ` "scope":"offiline_access" ` in every token. Was that desired?

Reply:
I added that scope to return the refresh token. I attach the documentation that I read to implement this: **[The offline access scope](https://learn.microsoft.com/en-us/entra/identity-platform/scopes-oidc#the-offline_access-scope)**